### PR TITLE
Added Ktlint

### PR DESCRIPTION
**Pull request by rozPierog**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,15 +1,16 @@
 matrix:
   include:
-    - language: objective-c
+    - language: objective-c, android
```
Comment (radex) [inline .travis.yml]: whoa, you can do that?

Comment (radex) [reply .travis.yml]: oh, apparently not :|

Comment (rozPierog) [reply .travis.yml]: Yeah, I was hoping this gonna work :|

**Comment by radex** [comment]
ooh! does it work @rozPierog ?

**Comment by radex** [comment]
Thank you @rozPierog for your patience with Travis 🙏 🎉

**Pull request merged**